joomla/joomla-cms
# [4.0] Remove unused file

Pull Request for Issue # .

### Summary of Changes

Removes `build/media_source/system/scss/_jalert.scss` which is unused.

### Testing Instructions

Run `node build/build.js --compile-css` and check that it completes without errors.

### Expected result AFTER applying this Pull Request

Works like before.

### Documentation Changes Required

No.
